translations/data/home.json

```diff
@@ -1,125 +1,143 @@
+	"de-de": "Fediverse ist ein Oberbegriff für soziale Medien, die zusammenarbeiten statt zu konkurrieren.",
         "sk": "Fediverse je sieť sociálnych médií, ktoré <mark>spolupracujú</mark> namiesto toho, aby súťažili.",
         "fr": "Le fediverse est un nom générique pour les réseaux sociaux qui <mark>coopèrent</mark>, plutôt que de se faire concurrence.",
         "ru": "Fediverse это общее название для социальных сетей, которые <mark>сотрудничают</mark> вместо того, чтобы конкурировать.",
         "ar": "الفديفيرس هو اسم شائع لمواقع التواصل الاجتماعي التي <mark> تتعاون </ mark> بدلاً من المنافسة."
     },
     "intro_fediverse_social_media": {
         "en-us": "And every major social media site has a great <mark>alternative</mark> in the fediverse, so it doesn't matter if you're into sharing images, videos, news, or just short quips. You will find <mark>your home</mark>.",
-        "sk": "A väčšina má skvelú <mark>alternatívu</mark> vo fediverse, takže nezáleží na tom, či chceš zdieľať obrázky, videá, správy, alebo len krátke vtipy. Nájdeš tu <mark>svoj domov</mark>.",
+	"de-de": "Und zu jeder wichtigen Social Media Seite gibt es im Fediverse tolle <mark>Alternativen</mark>. Es kommt also nicht darauf an, ob Du Bilder, Videos, Nachrichten oder witzige Sprüche austauschen möchtest. Du wirst ein passendes Zuhause finden.",
+	 "sk": "A väčšina má skvelú <mark>alternatívu</mark> vo fediverse, takže nezáleží na tom, či chceš zdieľať obrázky, videá, správy, alebo len krátke vtipy. Nájdeš tu <mark>svoj domov</mark>.",
         "fr": "Et chaque réseau social majeur dispose d'une super <mark>alternative</mark> dans le fediverse, donc peu importe que vous préfériez partager des images, des vidéos, des informations ou de courts messages. Vous trouverez ce qui <mark>vous convient le mieux</mark>.",
         "ru": "И каждая популярная социальная сеть имеет отличную <mark>альтернативу</mark> в fediverse, так что неважно, нравится ли вам делиться картинками, видео, новостями, или просто колкими шутками.",
         "ar": "وكل موقع من مواقع التواصل الاجتماعي الرئيسية لديه <mark> بديل </ mark> رائع في الفديفيرس، لذلك لا يهم إذا كنت تشارك الصور أو مقاطع الفيديو أو الأخبار أو مجرد المزح القصيرة. سوف تجد <mark> منزلك </ mark>."
     },
     "intro_btn_alright": {
         "en-us": "Alright!",
+	"de-de": "Zauberhaft !",
         "sk": "Super!",
         "fr": "Génial !",
         "ru": "Отлично!",
         "ar": "ممتاز!"
     },
     "outro_join": {
         "en-us": "Join the fediverse",
+	"de-de": "Komm ins Fediverse !",
         "sk": "Pripoj sa do fediversu",
         "fr": "Rejoindre le fediverse",
         "ru": "Присоединиться к fediverse",
         "ar": "انضم إلى الفديفيرس"
     },
     "outro_or": {
         "en-us": "or",
+	"de-de": "oder",
         "sk": "alebo",
         "fr": "ou",
         "ru": "или",
         "ar": "أو"
     },
     "outro_learn_more": {
         "en-us": "see how it works",
+	"de-de": "Schau Dir an, wie es funktioniert.",

```

Comment:
Either the sentence ending dot should be placed in all languages then, or removed here.

Reply:
Good point here, I think I'm just going to add dots everywhere to make this a proper sentence.

Reply:
@hoteit Quick question, do we need to add punctuation to the Arabic translation?

https://github.com/jointhefediverse-net/jointhefediverse.net/commit/7efe4ced6a6e9994568cb7039d916dc1a28a9db4